src/Execution/FieldCollector.php

```diff
@@ -106,6 +112,7 @@ protected function shouldIncludeNode(NodeInterface $node): bool
 
         $include = coerceDirectiveValues(IncludeDirective(), $node, $contextVariables);
 
+        /** @noinspection IfReturnReturnSimplificationInspection */
```

Comment:
I would change the return to a ternary instead

Reply:
I didn't want to do that because new execution paths could be introduced in the future.